ims-apple/IMS Apple/Utilities/Components/IMSTextField/IMSTextField.swift

```diff
@@ -10,6 +10,7 @@ import SwiftUI
 private enum Constants {
     static let textFieldMaxWidth: Double = 353
     static let textFieldMaxHeight: Double = 74
+    static let cornerRadius: CGFloat = 8
```

Comment:
I think a better name might be `cornerRadiusSize` because it is more declarative.

```suggestion
    static let cornerRadiusSize: CGFloat = 8
```